### PR TITLE
PR nhvaughan fork to origin

### LINKED ISSUE #12: Add image of liger
It's like my favorite animal.

### PR DESCRIPTION
**Pull request by nhvaughan**
Closes #12.

### CODE REVIEW

**Comment by nhvaughan** [comment]
Here I'm leaving a coment.

File: content/nhvaughan_first
Comment (nhvaughan) [inline content/nhvaughan_first:1]: Here I'm commenting on my first file

Comment (oshadura) [reply content/nhvaughan_first:1]: Congrats!

**Review by oshadura — APPROVED** [review]
Great job! Keep on working!

**Pull request merged**